bin/interpreter.sh

```diff
@@ -205,7 +214,9 @@ if [[ ! -z "$ZEPPELIN_IMPERSONATE_USER" ]]; then
 fi
 
 if [[ -n "${SPARK_SUBMIT}" ]]; then
-    if [[ -n "$ZEPPELIN_IMPERSONATE_USER" ]] && [[ "$ZEPPELIN_IMPERSONATE_SPARK_PROXY_USER" != "false" ]];  then
+    if [[ -n "${RUN_SPARK_ON_K8}" ]]; then
+       INTERPRETER_RUN_COMMAND+=' '` echo ${SPARK_SUBMIT} --class ${ZEPPELIN_SERVER} ${SPARK_SUBMIT_OPTIONS} --conf spark.app.name=zri-${INTERPRETER_GROUP_NAME} --conf spark.kubernetes.driver.label.interpreter-processId=${INTERPRETER_PROCESS_ID} --conf spark.metrics.namespace=zeppelin_${INTERPRETER_GROUP_NAME} ${SPARK_APP_JAR} ${PORT}`
```

Comment:
I think we need to match other mode to include `--driver-class-path \"${ZEPPELIN_INTP_CLASSPATH_OVERRIDES}:${ZEPPELIN_INTP_CLASSPATH}\" --driver-java-options \"${JAVA_INTP_OPTS}\" ${SPARK_SUBMIT_OPTIONS} ${ZEPPELIN_SPARK_CONF}`

Reply:
The issue is that these doesn't makes sense for the `apache-spark-on-k8s version of spark-submit`, beacuse it is launching the Spark driver in a separate pod, so any dependency should be passed via ResourceStagingServer or built into Spark Docker images.  

Currently you can't pass additional java options to Spark Driver, the supported options (https://apache-spark-on-k8s.github.io/userdocs/running-on-kubernetes.html) can be passed in SPARK_SUBMIT_OPTIONS.